src/ResourceManager/Websites/ChangeLog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Current Release
+* Added -AssignIdentity & -Httpsonly properties for Set-AzureRmWebApp
```

Comment:
@jocawtho this change log entry needs to be updated -- it looks like it was copied from one of the previous entries